tests/network/vmi_multus.go

```diff
@@ -407,7 +406,7 @@ var _ = SIGDescribe("[Serial]Multus", func() {
 
 				ipAddr := ""
 				if staticIPVm2 != "" {
-					ipAddr, err = cidrToIP(staticIPVm2)
+					ipAddr, err = libnet.CidrToIP(staticIPVm2)
```

Comment:
What is the wider usage ? Is anyone using this anywhere else ?

Reply:
I reconsidered; it makes sense to have this in the lib, rather than hidden away in a test file, and used by other tests.

Reply:
It is now used in multus, macvtap and sriov files.